.github/workflows/onPRClose.yml

```diff
@@ -21,3 +21,13 @@ jobs:
           issue-number: ${{ steps.commitMsgParser.outputs.issue-number }}
           label: 'in staging'
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  destroy:
```

Comment:
by the sound of the issue you only want to change the file name?

This block should belong in #3463 (if it's still applicable?)

Reply:
this is only a separate PR in order to get the github action to take the name change requested, it's the same piece of work as #3463. We can remove this block, merge this one, pull it back into #3463 and then make the change if you'd prefer

Reply:
yes please